2022-12-18 21:07:43,055 [INFO] [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2022-12-18 21:07:43,055 [INFO] [33mPress CTRL+C to quit[0m
2022-12-18 21:07:43,056 [INFO]  * Restarting with stat
2022-12-18 21:07:43,225 [WARNING]  * Debugger is active!
2022-12-18 21:07:43,238 [INFO]  * Debugger PIN: 386-252-342
2022-12-18 21:09:09,134 [INFO] [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2022-12-18 21:09:09,134 [INFO] [33mPress CTRL+C to quit[0m
2022-12-18 21:09:09,135 [INFO]  * Restarting with stat
2022-12-18 21:09:09,310 [WARNING]  * Debugger is active!
2022-12-18 21:09:09,324 [INFO]  * Debugger PIN: 386-252-342
2022-12-18 21:09:30,025 [INFO] [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2022-12-18 21:09:30,025 [INFO] [33mPress CTRL+C to quit[0m
2022-12-18 21:09:30,026 [INFO]  * Restarting with stat
2022-12-18 21:09:30,208 [WARNING]  * Debugger is active!
2022-12-18 21:09:30,226 [INFO]  * Debugger PIN: 386-252-342
2022-12-18 21:11:09,814 [INFO] [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2022-12-18 21:11:09,814 [INFO] [33mPress CTRL+C to quit[0m
2022-12-18 21:11:09,815 [INFO]  * Restarting with stat
2022-12-18 21:11:10,005 [WARNING]  * Debugger is active!
2022-12-18 21:11:10,018 [INFO]  * Debugger PIN: 386-252-342
2022-12-18 21:11:15,968 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:15] "GET /users/leo HTTP/1.1" 200 -
2022-12-18 21:11:16,015 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:16] "[36mGET /static/styles.min.css HTTP/1.1[0m" 304 -
2022-12-18 21:11:16,016 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:16] "[36mGET /static/img/bookmark.png HTTP/1.1[0m" 304 -
2022-12-18 21:11:16,016 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:16] "[36mGET /static/img/eye.png HTTP/1.1[0m" 304 -
2022-12-18 21:11:21,060 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:21] "GET /users/leo HTTP/1.1" 200 -
2022-12-18 21:11:21,065 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:21] "[36mGET /users/leo?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
2022-12-18 21:11:21,065 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:21] "[36mGET /users/leo?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2022-12-18 21:11:21,066 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:21] "[36mGET /users/leo?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
2022-12-18 21:11:21,093 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:21] "[36mGET /static/img/bookmark.png HTTP/1.1[0m" 304 -
2022-12-18 21:11:21,095 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:21] "[36mGET /static/styles.min.css HTTP/1.1[0m" 304 -
2022-12-18 21:11:21,097 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:21] "[36mGET /static/img/eye.png HTTP/1.1[0m" 304 -
2022-12-18 21:11:24,951 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:24] "[35m[1mGET /users/alex HTTP/1.1[0m" 500 -
2022-12-18 21:11:24,982 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:24] "GET /users/alex?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2022-12-18 21:11:24,983 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:24] "GET /users/alex?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2022-12-18 21:11:24,989 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:24] "GET /users/alex?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2022-12-18 21:11:25,052 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:25] "[36mGET /users/alex?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2022-12-18 21:11:46,336 [INFO] [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2022-12-18 21:11:46,336 [INFO] [33mPress CTRL+C to quit[0m
2022-12-18 21:11:48,595 [ERROR] Exception on /users/alex [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/main/views.py", line 28, in search_by_username
    posts = get_posts_by_user(username)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/utils.py", line 31, in get_posts_by_user
    raise ValueError('User does not exist')
ValueError: User does not exist
2022-12-18 21:11:48,597 [INFO] 127.0.0.1 - - [18/Dec/2022 21:11:48] "GET /users/alex HTTP/1.1" 200 -
2022-12-18 21:12:03,479 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:03] "GET / HTTP/1.1" 200 -
2022-12-18 21:12:03,511 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:03] "[36mGET /static/img/eye.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:03,513 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:03] "[36mGET /static/styles.min.css HTTP/1.1[0m" 304 -
2022-12-18 21:12:03,520 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:03] "[36mGET /static/styles.min.css HTTP/1.1[0m" 304 -
2022-12-18 21:12:03,520 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:03] "[36mGET /static/img/eye.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:03,523 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:03] "[36mGET /static/img/bookmark.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:03,524 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:03] "[36mGET /static/img/bookmark.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:05,803 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:05] "GET /templates/bookmarks.htmlmarks.html HTTP/1.1" 200 -
2022-12-18 21:12:08,702 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:08] "GET /static HTTP/1.1" 200 -
2022-12-18 21:12:15,873 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:15] "GET / HTTP/1.1" 200 -
2022-12-18 21:12:15,881 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:15] "[36mGET /static/styles.min.css HTTP/1.1[0m" 304 -
2022-12-18 21:12:15,882 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:15] "[36mGET /static/img/bookmark.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:15,882 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:15] "[36mGET /static/img/eye.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:15,896 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:15] "[36mGET /static/styles.min.css HTTP/1.1[0m" 304 -
2022-12-18 21:12:15,897 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:15] "[36mGET /static/img/bookmark.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:15,898 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:15] "[36mGET /static/img/eye.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:19,385 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:19] "GET /users/leo HTTP/1.1" 200 -
2022-12-18 21:12:19,390 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:19] "GET /users/leo?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2022-12-18 21:12:19,395 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:19] "[36mGET /static/styles.min.css HTTP/1.1[0m" 304 -
2022-12-18 21:12:19,395 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:19] "GET /users/leo?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2022-12-18 21:12:19,397 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:19] "GET /users/leo?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2022-12-18 21:12:19,399 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:19] "[36mGET /static/img/eye.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:19,401 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:19] "[36mGET /static/img/bookmark.png HTTP/1.1[0m" 304 -
2022-12-18 21:12:25,386 [ERROR] Exception on /users/alex [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/main/views.py", line 28, in search_by_username
    posts = get_posts_by_user(username)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/utils.py", line 31, in get_posts_by_user
    raise ValueError('User does not exist')
ValueError: User does not exist
2022-12-18 21:12:25,387 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:25] "GET /users/alex HTTP/1.1" 200 -
2022-12-18 21:12:25,391 [ERROR] Exception on /users/alex [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/main/views.py", line 28, in search_by_username
    posts = get_posts_by_user(username)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/utils.py", line 31, in get_posts_by_user
    raise ValueError('User does not exist')
ValueError: User does not exist
2022-12-18 21:12:25,391 [ERROR] Exception on /users/alex [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/main/views.py", line 28, in search_by_username
    posts = get_posts_by_user(username)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/utils.py", line 31, in get_posts_by_user
    raise ValueError('User does not exist')
ValueError: User does not exist
2022-12-18 21:12:25,392 [ERROR] Exception on /users/alex [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/main/views.py", line 28, in search_by_username
    posts = get_posts_by_user(username)
  File "/Users/dmitrijvinokurov/PycharmProjects/coursework2/utils.py", line 31, in get_posts_by_user
    raise ValueError('User does not exist')
ValueError: User does not exist
2022-12-18 21:12:25,392 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:25] "GET /users/alex?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2022-12-18 21:12:25,393 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:25] "GET /users/alex?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2022-12-18 21:12:25,394 [INFO] 127.0.0.1 - - [18/Dec/2022 21:12:25] "GET /users/alex?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2022-12-18 21:13:12,553 [INFO] [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2022-12-18 21:13:12,553 [INFO] [33mPress CTRL+C to quit[0m
2022-12-18 21:13:15,207 [INFO] 127.0.0.1 - - [18/Dec/2022 21:13:15] "GET /templates/bookmarks.htmlmarks.html HTTP/1.1" 200 -
2022-12-18 21:13:18,473 [INFO] 127.0.0.1 - - [18/Dec/2022 21:13:18] "GET /static HTTP/1.1" 200 -
2022-12-18 21:13:20,987 [INFO] 127.0.0.1 - - [18/Dec/2022 21:13:20] "GET /static HTTP/1.1" 200 -
